### PR TITLE
simulator: run all cases of the simulator in Makefile

### PR DESCRIPTION
**Pull request by rleungx**
## What have you changed? (required)
This PR is going to run all cases of simulator when excuting `make simulator`.

## What are the type of the changes (required)?

- Improvement (non-breaking change which is an improvement to an existing feature)

## How has this PR been tested (required)?

- Manual tests

### CODE REVIEW

File: cmd/simulator/main.go [hunk 1]
```diff
@@ -102,6 +116,25 @@ func NewSingleServer() (*server.Config, *server.Server, server.CleanupFunc) {
 		log.Fatalf("initialize logger error: %s\n", err)
 	}
 
+	lcfg := &zap.Config{
```
Comment (rleungx) [inline cmd/simulator/main.go]: I'm not sure if this is a proper way to mute the raft info log of etcd.

Comment (disksing) [reply cmd/simulator/main.go]: I think you can pass empty `OutputPaths` and `ErrorOutputPaths` if you want to disable all logs produced by raft. Please refer to https://godoc.org/go.uber.org/zap#Config and https://godoc.org/go.uber.org/zap#Open

BTW, I suppose you want to do it in main() instead reset it before running every case.

File: Makefile [hunk 2]
```diff
@@ -73,5 +73,6 @@ endif
 
 simulator:
 	CGO_ENABLED=0 go build -o bin/simulator cmd/simulator/main.go
+	bin/simulator
```
Comment (siddontang) [inline Makefile]: em I think mostly, we do build in Makefile, not include running it.

File: cmd/simulator/main.go (hunk 1 above)
Comment (nolouch) [reply cmd/simulator/main.go]: There is only one PD server. I think to disable the raft log is ok.

File: Makefile (hunk 2 above)
Comment (rleungx) [reply Makefile]: Any suggestion?

File: cmd/simulator/main.go [hunk 3]
```diff
@@ -43,8 +47,38 @@ var (
...
+		EncoderConfig: zap.NewProductionEncoderConfig(),
+
+		OutputPaths:      []string{},
+		ErrorOutputPaths: []string{},
+	}
+	lg, err := etcdlogutil.NewRaftLogger(lcfg)
+	if err != nil {
+		log.Fatalf("cannot create raft logger %v", err)
+	}
+	raft.SetLogger(lg)
```
Comment (disksing) [inline cmd/simulator/main.go]: Seems better to extract a function for disabling logs.

**Comment by disksing** [comment]
LGTM.

**Pull request merged**